runtime/src/accounts_background_service.rs

```diff
@@ -520,13 +519,12 @@ impl AbsRequestHandlers {
     // Returns the latest requested snapshot block height, if one exists
     pub fn handle_snapshot_requests(
         &self,
-        accounts_db_caching_enabled: bool,
+        _accounts_db_caching_enabled: bool,
```

Comment:
Is this removed in the next PR?

Reply:
yes. the fanout never stops. It is nice to do one signature at a time so you know all callers removed the 1st parameter, in this case. Sometimes parameters are (true, false, false, ...) and it is easy to get confused as to which one USED to be the enable_caching one.